pyInterface/decayAmplitude/isobarDecayTopology_py.cc

```diff
@@ -238,4 +248,13 @@ void rpwa::py::exportIsobarDecayTopology() {
 
 	bp::register_ptr_to_python<rpwa::isobarDecayTopologyPtr>();
 
+	bp::def(
+		"createIsobarDecayTopology"
+		, &createIsobarDecayTopology_1
+	);
+	bp::def(
+		"createIsobarDecayTopology"
+		, &createIsobarDecayTopology_2
+	);
```

Comment:
Is this really the correct order to specify the functions, I seem to remember that the first function defined is the last function that is tried, so the first defined function should be the more general. Did you make sure that the correct functions are called for both arguments?